src/posts/features-and-projects.md

```diff
@@ -0,0 +1,146 @@
+
+![web-mcdu](/img/notam-images/march2022/mcdu-day-night.png)
+
+We have used this first development as a means of evaluating our strategy regarding custom instruments. We are currently working on developing custom high-speed protocols for offloading instrument rendering onto other programs or even other devices, over a network, to offer better possibilities for home cockpit integration.
+
+### Nose Wheel Tiller
+We implemented a way to use the tiller independently of the rudder utilising custom hydraulic steering. Based on our QA tests and feedback from our type rated pilots, this is a great start that helps accurately represent realistic situations when taxiing the aircraft. You can read more about how to use this feature and associated information in the guide below.
+
+[Nose Wheel Tiller Guide](https://docs.flybywiresim.com/fbw-a32nx/feature-guides/nw-tiller/)
+
+### Stored Waypoints
+We also implemented stored waypoints recently allowing pilots to enter in custom waypoints to expand your data management for your flights. This allows for pilots to create, enter, and utilise pre-planned waypoints with a designated name such as “MYWP1” without having to type in latitude/longitude coordinates mid-flight.
+You can read more about how to use this feature and associated information in the guide below.
+
+[Stored Waypoints Guide](https://docs.flybywiresim.com/pilots-corner/advanced-guides/data-management/)
+
+![Stored Waypoints](/img/notam-images/march2022/stored-waypoints.png "Stored Waypoints")
+
+### flyPadOS v3
+We have been working hard on a new version of the flyPad featuring a beautiful new design and many additional requested features.
+
+- Better navigation handling remembering previously visited tabs and input fields
+- Ability to pin charts
+- Checklists
+- Enhanced pushback features
+- and much more!
+
+Below is a preview of the upcoming update. (Work in progress and final layout may look different.)
+
+![efb-checklist](/img/notam-images/march2022/efb-checklist.png)
+
+![efb-atc](/img/notam-images/march2022/efb-atc.png)
+
+![efb-charts](/img/notam-images/march2022/efb-charts.png)
+
+![efb-dashboard](/img/notam-images/march2022/efb-dashboard.png)
+
+---
+
+## Other Projects
+
+### Discord Bot
+
+The FlyByWire Discord bot has been updated to discord.js v13 and API v9. As a result of this update the bot now works in threads, making it even more of an invaluable tool for our Community Support Team to help users diagnose issues! While there may not be many changes to the functionality of the bot, we have laid the foundation to add many great features in the future! 
+
+Now that the update is complete, we have some exciting projects in the pipeline that we have started to work on - one of them being music integration! It would be beneficial for more people to join the Bot Development Team, so we can keep up with the regular workload of adding and updating commands to be used across the Discord server!
```

Comment:
```suggestion
Now that the update is complete, we have some exciting projects in the pipeline that we have started to work on: music integration! It would be beneficial for more people to join the Bot Development Team to keep up with the regular workload of adding and updating commands to be used across the Discord server!
```